src/gallia/services/uds/server.py

```diff
@@ -899,6 +900,7 @@ async def run(self) -> None:
                     await transport.write(uds_response_raw)
             except Exception:
                 traceback.print_exc()
+                break
```

Comment:
Currently this kills the ISOSTPServer since there is no recovery.

What exceptions do you expect that are recoverable, @fkglr ?

Reply:
`ISOTPUDSServerTransport` now dies after a single error, but let's see whether that becomes a problem and handle it once it occurs.